src/main/java/view/AssignmentAddView.java

```diff
@@ -74,17 +74,13 @@ public void actionPerformed(ActionEvent evt) {
                         if (evt.getSource().equals(addAssignment)) {
                             final AddAssignmentState currentState = addAssignmentViewModel.getState();
 
-                            try {
```

Comment:
The removal of the try-catch here is causing code conflicts – see if you're able to fix this on your end or in Github